go/install/install.go

```diff
@@ -273,3 +276,41 @@ type CommonLsofResult struct {
 	PID     string
 	Command string
 }
+
+func fileContainsWord(filePath, searchWord string) bool {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return false
+	}
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		if strings.Contains(scanner.Text(), searchWord) {
+			return true
+		}
+	}
+	return false
+}
+
+func LastModifiedMatchingFile(dirPath string, fileNameMatch string, fileContentMatch string) (res *string) {
+	allFiles, err := ioutil.ReadDir(dirPath)
+	if err != nil {
+		return nil
```

Comment:
log error. Caller uses `fmt.Fprintf(logFile, "...\n")`

Reply:
this is logging to the file that hasn't actually been opened yet. i did some moving around though, and now the errors from all of these bubble up and get printed out into the file when it does get opened.